tech/languages/php/php-installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: PHP
+page: php
+section: tech-languages
+---
+
+## PHP
+
+### PHP Installation
+
+To install PHP, simply type:
+
+```
+# dnf install php-cli
+```
+
+Above command will install latest stable PHP package, which includes the command-line interface executing PHP scripts, /usr/bin/php, and the CGI interface. 
+
+
+You might want to install some other packages, e.g. phpunit for unit tests or composer to manage dependencies of PHP projects. You can install them with:
+
+```
+# dnf install phpunit composer
+```
+
+### PHP modules
+PHP provides only a few set of extensions, other should be installed separately. For example, if you need "mysqli" extension, use:
+
+```
+# dnf install php-mysqli
```

Comment:
Is every PHP extension packaged named php-NAME? If so, please write it the way this is clear. Is there also some virtual provide users can use to find/install PHP extensions?